src/app/scoreboard/stub/score.js

```diff
@@ -0,0 +1,46 @@
+import { isUndefined } from 'util'
+
+/**
+ * @name parseScore
+ * @desc `fetch()` scoreboard
+ * @param {Function} func - Function to call when window.hestia.contest.scoreboard has been updated
+ * @returns {Promise} a `fetch()` `Promise`
+ */
+
+async function parseScore(func) {
+    return fetch(
+        `http://${window.location.hostname}:${window.location.port}/api/score`
```

Comment:
Shorten this to "/api/score"

Reply:
Address in https://github.com/vnma0/hestia/pull/19/commits/84c0282afababde0e3ad70cf90345d947d22553c.